### PR TITLE
feat: up richContainer data get & gi-richcontainer-side  minWidth limit add

### PR DESCRIPTION
**Pull request by noneAuth**
### 👀 PR includes

✨ Feature

- [ 1] New feature
 update  gi-rich-container-toolbar => toolbarItem click default checkded date model

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [1 ] Change the UI
   action: add gi-richcontainer-side minWidth (336px) model
   why: this action solves the problem of distorted display with a view width that is too small 
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌     | ✅    |

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: pnpm-workspace.yaml [hunk 1]
```diff
@@ -2,5 +2,5 @@ packages:
   # all packages in subdirs of packages/ and components/
   - 'packages/**'
   - '!packages/gi-cli/templates'
-  - '!packages/gi-assets-tugraph-db'
+  # - '!packages/gi-assets-tugraph-db'
```
Comment (lxfu1) [inline pnpm-workspace.yaml:5]: 这里reset掉，无效改动

File: pnpm-lock.yaml
Comment (lxfu1) [inline pnpm-lock.yaml:1]: 无效改动？

File: packages/gi-site/src/services/inject.ts [hunk 2]
```diff
@@ -1 +1,2 @@
-export default {};
\ No newline at end of file
+import * as GI_ASSETS_TUGRAPH_DB from '@tugraph/gi-assets-tugraph-db';
+export default { GI_ASSETS_TUGRAPH_DB };
```
Comment (lxfu1) [inline packages/gi-site/src/services/inject.ts:2]: gi-site 应该不需要改

File: pnpm-workspace.yaml (hunk 1 above)
Comment (noneAuth) [reply pnpm-workspace.yaml:5]: done

File: packages/gi-site/src/services/inject.ts (hunk 2 above)
Comment (noneAuth) [reply packages/gi-site/src/services/inject.ts:2]: done

File: pnpm-lock.yaml
Comment (noneAuth) [reply pnpm-lock.yaml:1]: done

**Pull request closed without merging**